vmdb/lib/workers/worker_base.rb

```diff
@@ -150,10 +151,15 @@ def log_prefix
   #
 
   def find_worker_record
-    @worker = self.class.corresponding_model.find_by_guid(@cfg[:guid])
+    @worker = self.class.corresponding_model.includes(:miq_server).find_by_guid(@cfg[:guid])
     do_exit("Unable to find instance for worker GUID [#{@cfg[:guid]}].", 1) if @worker.nil?
   end
 
+  def find_server_record
+    find_worker_record unless @worker
+    @server = @worker.miq_server
```

Comment:
Maybe we should leverage the worker reader method instead of the ivar like we do all over the place?

Should we create server attr_reader or attr_accessor?